### PR TITLE
Add Karlsruher Institut für Technologie

### PR DESCRIPTION
**Pull request by paulexyz**

### CODE REVIEW

**Comment by paulexyz** [comment]
I don't understand why the Travis CI build fails

File: _data/education.yml [hunk 1]
```diff
@@ -247,6 +247,14 @@ websites:
       twitter: ISACANews
       facebook: ISACAHQ
 
+   - name: Karlsruher Institut für Technologie
```
Comment (kmpoppe) [inline _data/education.yml]:
```suggestion
    - name: Karlsruher Institut für Technologie
```
Travis can't find the start tag, when the yaml syntax isn't adhered to.

Comment (kmpoppe) [reply _data/education.yml]: @PauleTV As you have not set your pull request to write access by members, I'd kindly ask you to commit my suggestion. Thanks!

**Review by kmpoppe — DISMISSED** [review]

File: _data/education.yml [hunk 2]
```diff
@@ -423,4 +431,4 @@ websites:
       phone: Yes
       software: Yes
       hardware: Yes
-      doc: https://www.tech.it.vt.edu/2factor/
+      doc: https://www.tech.it.vt.edu/2factor/
```
Comment (Carlgo11) [inline _data/education.yml]:
```suggestion
      doc: https://www.tech.it.vt.edu/2factor/
```

**Review by Carlgo11 — DISMISSED** [review]
Looks like you accidentally edited line 434.

File: _data/education.yml (hunk 2 above)
Comment (kmpoppe) [reply _data/education.yml]: @paulexyz bump

Comment (kmpoppe) [reply _data/education.yml]: @paulexyz bump!

Comment (paulexyz) [reply _data/education.yml]: *fixed*

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**